src/patternfly/components/Pagination/pagination.scss

```diff
@@ -28,18 +28,31 @@
     white-space: nowrap;
   }
 
-  > *:not(:last-child):not(.pf-c-pagination__total-items) {
+  > *:not(:last-child):not(.pf-c-pagination__items) {
     margin-right: var(--pf-c-pagination--child--MarginRight);
   }
 
   // Top pagination
   &:not(.pf-m-footer) {
-    .pf-c-options-menu,
     .pf-c-pagination__nav {
-      display: none;
+      display: flex;
 
-      @media screen and (min-width: $pf-global--breakpoint--md) {
-        display: flex;
+      @media screen and (max-width: $pf-global--breakpoint--md) {
+        display: none;
+        visibility: hidden;
+      }
+    }
+  }
+
+  &.pf-m-footer {
+    margin-left: auto;
```

Comment:
is this necessary? I don't see what it's doing.

Reply:
When I put pagination in the toolbar (table and list demo)  it defaults to a left alignment, so this fixes that.

Reply:
Shouldn't the toolbar be responsible for positioning it's elements?

Reply:
Yes updated